community/index.md

```diff
@@ -43,6 +43,7 @@ Various Julia projects are hosted under the following umbrella organizations on
 * [JuliaCI](https://github.com/JuliaCI) – Continuous Integration Support for Julia packages
 * [JuliaDSP](https://github.com/JuliaDSP) – Digital signal processing
 * [JuliaGraphs](https://github.com/JuliaGraphs) – Graph Theory and Implementation
+* [JuliaMath](https://github.com/JuliaMath) -.Mathematics made easy in Julia
```

Comment:
Thanks for catching this.
Its not actually an extra `.` causing a problem, so much as a `-` vs `–`
(I think hyphen vs en-dash) 
(odd that it is rendering differently the the coder in you line comment, or is it? Everyline has a `–.` in the git editor that i can see, but not in the diff...)